packages/types/util.d.ts

```diff
@@ -118,15 +118,13 @@ type Intersect<U> = (U extends any ? (k: U) => void : never) extends (
   ? I
   : never
 
-export type Exclusive<T> = keyof T extends infer Keys
-  ? Keys extends infer Key
+export type Exclusive<T> = AllKeys<T> extends infer K
```

Comment:
@joshuaellis what is `AllKeys`? My project fails as this doesn't exist?

Reply:
That's odd. I don't understand why `tsc` test would pass if `AllKeys` if undefined.

Reply:
new version 9.5.4 released, 9.5.3 has been deprecated.